### PR TITLE
TSK-56 hey

### PR DESCRIPTION
**Pull request by dkochiashvili**
TSK-56 this is demo PR.

### CODE REVIEW

File: inga.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+gamarjoba
```
Comment (ParthGalen) [inline inga.txt:1]: change it to georgian

Comment (IrinaGomareli) [inline inga.txt:1]: it's ok

**Comment by erukhadze** [comment]
it's great

File: inga.txt (hunk 1 above)
Comment (dkochiashvili) [reply inga.txt:1]: but, I'll try change.

Comment (ParthGalen) [reply inga.txt:1]: no, it's not :)

**Review by ParthGalen — CHANGES_REQUESTED** [review]
do it )

File: inga.txt (hunk 1 above)
Comment (dkochiashvili) [reply inga.txt]: I did it.

**Review by ParthGalen — APPROVED** [review]
good!

File: inga.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+გამარჯობა.
```
Comment (erukhadze) [inline inga.txt:1]: good job

**Review by ParthGalen — APPROVED** [review]

**Review by IrinaGomareli — APPROVED** [review]

**Review by IrinaGomareli — APPROVED** [review]

**Review by erukhadze — APPROVED** [review]

**Pull request merged**